Resources/Textures/Objects/Specific/Mech/hamtr_construction.rsi/meta.json

```diff
@@ -0,0 +1,101 @@
+{
+  "copyright" : "Taken from https://github.com/tgstation/tgstation at at https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a",
```

Comment:
i thought you drew these from scratch

Reply:
yea i did oops forgot to change the copyright when i pasted json from ripley :D

Reply:
should be fixed